site/en/blog/webgpu-release/index.md

```diff
@@ -0,0 +1,56 @@
+3000 issues](https://github.com/gpuweb/gpuweb/graphs/contributors)), from the initial design in 2017, the first implementation is now available in Chrome, with support for Firefox and Safari in progress.
+
+Both the [Dawn](https://dawn.googlesource.com/dawn) library for Chromium and the [wgpu](https://github.com/gfx-rs/wgpu) library for Firefox are available as standalone packages, and they offer great portability and ergonomic layers that abstract OS GPU APIs. Using these libraries in native applications also makes it easier to port to [WASM](https://webassembly.org/) through [Emscripten](https://emscripten.org/) and [Rust web-sys](https://rustwasm.github.io/wasm-bindgen/web-sys/index.html).
+
+## Browser support
+
+This initial release of WebGPU is available in Chrome&nbsp;113 on ChromeOS devices with Vulkan support, Windows devices with Direct3D 12 support, and macOS. Linux, Android, and expanded support for existing platforms will come soon.
+
+WebGPU is currently a [work-in-progress](https://github.com/gpuweb/gpuweb/wiki/Implementation-Status) in [Firefox](https://hacks.mozilla.org/2020/04/experimental-webgpu-in-firefox/) and [Safari](https://github.com/WebKit/WebKit/commits/main/Source/WebCore/Modules/WebGPU), in addition to the initial implementation in Chrome.
+
+## Library support
+
+Many widely used WebGL libraries are already in the process of implementing WebGPU support or have already done so. This means that using WebGPU may only require making a single line change:
+- [Babylon.js](https://babylonjs.com/) has full WebGPU support already, see [demo](https://playground.babylonjs.com/#YX6IB8).
+- [PlayCanvas](https://blog.playcanvas.com/initial-webgpu-support-lands-in-playcanvas-engine-1-62/) announced initial WebGPU support.
+- [TensorFlow.js](TODO) supports WebGPU-optimized versions of most operators
+- [Three.js](https://threejs.org) WebGPU support is underway, see [examples](https://threejs.org/examples/?q=webgpu#webgpu_particles).
+
+## Resources
+
+WebGPU is a significant technology, and we recommend the following resources to learn more:
+- Check out the W3C specifications for [WebGPU](https://gpuweb.github.io/gpuweb/) and [WGSL](https://gpuweb.github.io/gpuweb/wgsl/).
+- Experiment with [samples](https://webgpu.github.io/webgpu-samples/) and explore WGSL with a [tour](https://google.github.io/tour-of-wgsl/).
+- Look at the [MDN documentation](https://developer.mozilla.org/docs/Web/API/WebGPU_API).
+- Read the official [explainer](https://gpuweb.github.io/gpuweb/explainer/) and [best practices](https://toji.dev/webgpu-best-practices/).
+- Learn about [GPU compute](/articles/gpu-compute/) and [more](https://surma.dev/things/webgpu/index.html).
+
+Many thanks to all Chromium contributors and especially to Intel folks for their invaluable support in making this possible.
```

Comment:
@rachelandrew Do you mind if I add a `## Acknowledgements` section just before this line so that it's clear it's not part of Resources section above?

Reply:
Yep that's fine.

Reply:
Done